web/widgets.py

```
@@ -135,3 +139,224 @@ def __init__(self, attrs=None):
+class TailwindJSONWidget(forms.Widget):
+    """A custom widget for editing JSON arrays of strings as a list of inputs with drag-and-drop reordering."""
+
+    def __init__(self, attrs: Dict = None) -> None:
+        default_attrs = {
+            "class": "json-features-widget",
+        }
+        if attrs:
+            default_attrs.update(attrs)
+        super().__init__(default_attrs)
+
+    def render(self, name: str, value: Any, attrs: Dict = None, renderer=None) -> str:
```

Ruff (0.8.2)

Dynamically typed expressions (typing.Any) are disallowed in `value`

(ANN401)